src/components/Card.jsx

```diff
@@ -9,6 +9,8 @@ export default function Card({
   expirationDateMM = "",
   expirationDateYY = "",
   cardOwnerName = "",
+  ...props
```

Comment:
c: `CardBox`에 `HTMLDivElement`의 속성들을 전달하기 위한 rest 구문으로 보이네요. 타입스크립트가 아니다보니 나중에는 어떤 인자들인지 파악이 어려울 것 같습니다. 각각의 prop을 명시적으로 작성하셔야 휴먼에러로 인한 버그를 방지할 수 있을 것 같아요.

```suggestion
  onClick
```